test/e2etest/bitbucket_test.go

```diff
@@ -22,6 +22,9 @@ func bitbucketHealthTests(t *testing.T, testConfig TestConfig, productUrl string
 	assertBitbucketStatusEndpoint(t, productUrl)
 	assertBitbucketNfsConnectivity(t, testConfig)
 	assertBitbucketSshConnectivity(t, testConfig, productUrl)
+	// give Bitbucket enough time to create project and repo indexes
+	time.Sleep(15 * time.Second)
```

Comment:
To keep things clean, can this sleep be moved into the function `assertEsIndexes` and be the firs thing it does.